### ISSUE TITLE
[Bug] NFTs Progress cannot be closed

### ISSUE THREAD

**Issue by ayush-that** [Avocodos/avocodos#20]
![{194BFDCB-18A6-4F9C-A1DF-0146E692F943}](https://github.com/user-attachments/assets/d20c562d-fb5f-4d5a-932b-6fa4c42e8a02)

Once I open the NFTs Progress bar, I'm unable to close it unless I press the back button. There is no cross button. Clicking off the popup does not close it as well.

**Comment by HarjjotSinghh** [comment]
Yes, this is supposed to be intentional - a bit annoying I get it, but this is only the case when `?showRewards=true` is set in the URL (you get redirected here from your notifications).

Do you think we should add a close button in this menu? I would love to hear your thoughts about the same.

**Comment by ayush-that** [comment]
> Yes, this is supposed to be intentional - a bit annoying I get it, but this is only the case when `?showRewards=true` is set in the URL (you get redirected here from your notifications).
> 
> Do you think we should add a close button in this menu? I would love to hear your thoughts about the same.

Yes, please. Add a close button or an exit route from this state.
